### DISCUSSION TITLE
Missing msopenjdk in Microsoft repository for SLES 15

### DISCUSSION THREAD

**Discussion by ferben** [microsoft/openjdk#583] [Q&A] [answered]
There are:
1. msopenjdk-11.x86_64 : Microsoft Build of OpenJDK 11
2. msopenjdk-16.x86_64 : OpenJDK Development Kit 16 (JDK) with Hotspot by Microsoft
3. msopenjdk-17.x86_64 : Microsoft Build of OpenJDK 17
4. msopenjdk-21.x86_64 : Microsoft Build of OpenJDK 21 

in Microsoft repo for CentOS7/RHEL7 (baseurl=https://packages.microsoft.com/centos/7/prod)

But in repository for SLES 12 or 15 any msopenjdk missing, why?

**Comment by karianna** [comment]
Hi @ferben - We do not currently provide installers for SLES. What is your use case here?  Are you running SLES on a VM in Azure for example?

**Comment by ferben** [comment]
Hello Martijn,
we would like provide our multi-platform solution with msopenjdk.
We support RHEL+SLES (as rpm-based distros), so we'd like to use one JDK on all platforms.
Unfortunately SUSE does not provide their own openjdk yet.
This is only proposal/requirement.

Thank you

**Reply by karianna** [reply]
Thanks will discuss with the Product Managers and see if there's other demand for this as well.

**Reply by karianna** [reply, marked as answer]
We have added this to our backlog for consideration for the April 2024 PSU release (this is not a concrete promise at this stage but we understand the work that needs to be done).
